.travis.yml

```diff
@@ -4,6 +4,7 @@ dist: trusty
 arch:
   - amd64
   - arm64
+  - s390x
```

Comment:
any interest in adding ppc64le while you are in here? 😇

Reply:
This looks surprisingly easy, so maybe it's a simple enough follow up. 🤔

Reply:
good point, will check if it works for Power :)

btw, you are doing so many great things for arm64 in this direction :) Could you also take into account s390x and ppc64le then 😊  ?

Reply:
@mattmoor ppc64le cross compilation works. So I added it to the PR

Reply:
> Could you also take into account s390x and ppc64le

Generally I am trying to anticipate a proliferation of architectures as I'm expanding the support for things, but I'm a bit reluctant to make changes where I don't have hardware to test and abusing automated testing is impractical (e.g. cases using bazel or kind).  For `ko` I did add `ko publish` testing last Friday for all four of the distroless architectures, but in general these are "just" 4 of a much longer list 😉